Wangscape/options_schema.json

```diff
@@ -180,13 +179,22 @@
           "Terrains"
         ]
       }
+    },
+    "AlphaCalculatorMode": {
+      "type":"string",
+      "title":"AlphaCalculatorMode schema",
+      "description":"Specifies which type of AlphaCalculator should be used to convert corner weights into alpha values.",
+      "enum":["Max",
+              "Linear"]
+              
     }
   },
   "required": [
     "TileFormat",
     "OutputDirectory",
     "Terrains",
     "Cliques",
+    "AlphaCalculatorMode",
     "BorderOptions"
   ]
-}
\ No newline at end of file
+}
```

Comment:
I've created [an issue](https://github.com/Wangscape/Wangscape/issues/79) with moving files like this to `doc/`.